crates/net/eth-wire-types/src/message.rs

```diff
@@ -252,6 +262,7 @@ impl<N: NetworkPrimitives> EthMessage<N> {
             Self::NodeData(_) => EthMessageID::NodeData,
             Self::GetReceipts(_) => EthMessageID::GetReceipts,
             Self::Receipts(_) => EthMessageID::Receipts,
+            Self::UpgradeStatus(_) => EthMessageID::UpgradeStatus,
```

Comment:
we technically don't need these here because  this is only exchanged in the handshake so we should be able to get away without the variant